### ISSUE TITLE
Edit firebase alert on incorrect/correct login

### ISSUE THREAD

**Issue by isha-73** [narayan954/dummygram#277]
### Description

Once the login details are detected incorrect, the alert is not in Leyman terms. Its like "Firebase: The email address is badly formatted. (auth/invalid-email)". So I want to work on this issue. Please assign me this.

### Screenshots

![image](https://github.com/narayan954/dummygram/assets/87441080/d09b7c43-555c-4cd3-aaed-9ae07fc7f42f)

**Comment by narayan954** [comment]
> ### Description
> Once the login details are detected incorrect, the alert is not in Leyman terms. Its like "Firebase: The email address is badly formatted. (auth/invalid-email)". So I want to work on this issue. Please assign me this under GSSOC'23.
> 
> ### Screenshots
> ![image](https://user-images.githubusercontent.com/87441080/239721062-d09b7c43-555c-4cd3-aaed-9ae07fc7f42f.png)
> 
> ### Additional information
> _No response_

Hi there @isha-73 , can you solve this bug using client side validation as this error should be handled at the client side itself and shouldn't reach the server side (firebase) which is the cause of that error?

Regards

**Comment by isha-73** [comment]
@narayan954 yes it is client side bug only, I can do it.. can u please assign it to me?

**Comment by narayan954** [comment]
> @narayan954 yes it is client side bug only, I can do it.. can u please assign it to me?

Yeah sure, you can go ahead! :)

**Comment by programerr01** [comment]
hii @narayan954  , assign it to me as the original author doesn't seem to be working on it

**Comment by narayan954** [comment]
> hii @narayan954 , assign it to me as the original author doesn't seem to be working on it

Yeah sure, go ahead! :)

**Comment by isha-73** [comment]
@narayan954 I will do it give me some time as I was busy in some other work...

**Comment by narayan954** [comment]
> @narayan954 I will do it give me some time as I was busy in some other work...

@isha-73 It's already been 2 weeks. Please do it asap or before tomorrow night else @programerr01 will get the assignment.

**Comment by narayan954** [comment]
> hii @narayan954 , assign it to me as the original author doesn't seem to be working on it

Hi @programerr01 , let's allow her till tomorrow, then you can work on this. Sorry for the inconvenience.
